core/src/test/java/dev/streamx/cli/run/MeshDefinitionResolverTest.java

```diff
@@ -78,34 +78,18 @@ void shouldThrowExceptionIfThereIsNoMeshInCurrentDirectory() throws IOException
     assertThat(exception).hasMessage("Mesh definition not found");
   }
 
-  @Test
-  void shouldResolveBlueprintPredefined() throws IOException {
-    // given
-    MeshSource meshSource = new MeshSource();
-    meshSource.blueprintsMesh = true;
-
-    // when
-    MeshDefinition result = uut.resolve(meshSource);
-
-    // then
-    assertNotNull(result);
-    assertThat(result.path()).isNull();
-    assertThat(result.serviceMesh()).isNotNull();
-  }
-
   @Test
   void shouldResolveGivenMeshDefinition() throws IOException {
     // given
     MeshSource meshSource = new MeshSource();
-    meshSource.blueprintsMesh = false;
-    meshSource.meshDefinitionFile = "target/classes/streamx-mesh.yml";
+    meshSource.meshDefinitionFile = "target/test-classes/streamx-mesh.yml";
```

Comment:
this path is hardcoded several times. Cant use use classloader to load resource by name ?  Or at least make a const